### PR TITLE
Feat/quickswap

### PR DESCRIPTION
**Pull request by cryptoDev222**
- [x] Review and approve connector @pradyuman-verma 
- [x] Deploy the contracts and comment the addresses @pradyuman-verma 
- [x] Created resolver - [here](https://github.com/Instadapp/dsa-resolvers/pull/16)
- [x] Add to the dsa-connect - [here](https://github.com/Instadapp/dsa-connect/pull/112)
- [ ] Add to the docs - link the PR
- [ ] integrate on main app - link PR

### CODE REVIEW

File: contracts/polygon/connectors/quickswap/helpers.sol [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+import {DSMath} from "../../common/math.sol";
+import {Basic} from "../../common/basic.sol";
+import {IQuickSwapRouter, IQuickSwapFactory} from "./interface.sol";
+
+abstract contract Helpers is DSMath, Basic {
+    /**
+     * @dev IQuickSwapRouter
+     */
+    IQuickSwapRouter internal constant router =
+        IQuickSwapRouter(0xd9e1cE17f2641f24aE83637ab66a2cca9C378B9F);
```
Comment (pradyuman-verma) [inline contracts/polygon/connectors/quickswap/helpers.sol]: The quickswap router address on polygon chain is - `0xa5E0829CaCEd8fFDD4De3c43696c57F7D7A678ff`

**Comment by pradyuman-verma** [comment]
Sushiswap tests are passing: 
image

File: contracts/polygon/connectors/quickswap/helpers.sol (hunk 1 above)
Comment (pradyuman-verma) [reply contracts/polygon/connectors/quickswap/helpers.sol]: commited

**Comment by pradyuman-verma** [comment]
SushiIncentive tests:
image

**Comment by pradyuman-verma** [comment]
Quickswap test on polygon: 
image

**Comment by pradyuman-verma** [comment]
##Deployed Quickswap Connector:

**chain:** Polygon
**connectorName:** "QUICKSWAP-A",
**connectorAddress:** [0x7B00D124Fc98107ff97fAc5d37493E1784Bb5C8E](https://polygonscan.com/address/0x7B00D124Fc98107ff97fAc5d37493E1784Bb5C8E)

**Review by KaymasJain — APPROVED** [review]

**Pull request merged**